src/TBuffer.cpp

```diff
@@ -2430,11 +2430,20 @@ TBuffer TBuffer::copy(QPoint& P1, QPoint& P2)
     if ((x < 0) || (x >= static_cast<int>(buffer.at(y).size())) || (P2.x() < 0) || (P2.x() > static_cast<int>(buffer.at(y).size()))) {
         x = 0;
     }
-
+    int oldLinkId, id = 0;
```

Comment:
```suggestion
    int oldLinkId{}, id{};
```

SlySven is right, see https://www.learncpp.com/cpp-tutorial/variable-assignment-and-initialization/ for explanation

Reply:
Oh, going the fancy, new and shiny C++11 (apparently) brace `{}` initialiser route are we? I still ain't totally at ease with these new-fangled (only 10 year old) way... :wink: